### PR TITLE
OCM-11069 | test: Automate OCP-38780 and OCP-38783

### PR DESCRIPTION
**Pull request by OAharoni-RedHat**
```
oaharoni-mac:rosa oaharoni$ ginkgo run -focus 38783 tests/e2e
Running Suite: ROSA CLI e2e tests suite - /Users/oaharoni/rosa_workspace/rosa/tests/e2e
=======================================================================================
Random Seed: 1726156603

Will run 1 of 224 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 224 Specs in 3.531 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 223 Skipped
PASS

Ginkgo ran 1 suite in 9.275899125s
Test Suite Passed
```

```
oaharoni-mac:rosa oaharoni$ ginkgo run -focus 38780 tests/e2e
Running Suite: ROSA CLI e2e tests suite - /Users/oaharoni/rosa_workspace/rosa/tests/e2e
=======================================================================================
Random Seed: 1726156677

Will run 1 of 224 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 224 Specs in 4.340 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 223 Skipped
PASS

Ginkgo ran 1 suite in 10.13421825s
Test Suite Passed
```

### CODE REVIEW

File: tests/e2e/test_rosacli_ingress.go [hunk 1]
```diff
@@ -508,3 +508,63 @@ var _ = Describe("Edit ingress",
...
+			labels.Medium, labels.Runtime.Day2,
+			func() {
+				if isHosted {
+					SkipNotClassic()
+				}
+				By("Fail to delete ingress when no ingress or cluster are passed")
+				output, err := ingressService.DeleteIngress("", "")
+				Expect(err).To(HaveOccurred())
+				Expect(output.String()).Should(
+					ContainSubstring("Expected exactly one command line parameter containing the id of the ingress"))
```
Comment (radtriste) [inline tests/e2e/test_rosacli_ingress.go:548]: Why is that not `required flag(s) \"cluster\" not set` ?

File: tests/utils/exec/rosacli/ingress_service.go [hunk 2]
```diff
@@ -134,9 +134,17 @@ func (i *ingressService) ReflectIngressList(result bytes.Buffer) (res *IngressLi
 
 // Delete the ingress
 func (i *ingressService) DeleteIngress(clusterID string, ingressID string) (output bytes.Buffer, err error) {
+	var flags []string
```
[the remark this answers is not in the document]
Comment (OAharoni-RedHat) [reply tests/utils/exec/rosacli/ingress_service.go:137]: good call! Changed

File: tests/e2e/test_rosacli_ingress.go (hunk 1 above)
Comment (OAharoni-RedHat) [reply tests/e2e/test_rosacli_ingress.go:548]: Hmm I guess I don't know which one I think makes more sense to prioritize in this case. Do we generally show that error with top priority? If so I will cut a ticket for the developers. In the mean time I've added another case to confirm that the `"cluster" not set` error does occur when an Ingress is passed but not a cluster

**Pull request closed without merging**